oap-server/analyzer/agent-analyzer/src/test/resources/trace-sample-rate-setting.yml

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+default:
+  sampleRate: 10000
+  duration: -1
+services:
+  - name: name1
+    rate: 1000
+    duration: 20000
+  - name: name2
+    rate: 2000
+    duration: 30000
```

Comment:
- Why use `sampleRate` for `default` and `rate` for `services`? I think it better to be consistent.
- I think the following format might be also more consistent.

```yaml
default:
  sampleRate: 10000
  duration: -1
service1:
  sampleRate: 1000
  duration: 20000
name2:
  sampleRate: 2000
  duration: 30000 
```